Resources/Prototypes/_CD/Entities/Clothing/Eyes/hud.yml

```diff
@@ -0,0 +1,13 @@
+- type: entity
+  parent: [ClothingEyesBase, ShowMedicalIcons]
+  id: ClothingEyesHudMedicalVisor
+  name: medical visor
+  description: A full-vision medical hud, allowing more complex readouts for diagnostics.
```

Comment:
I think this description implies that it does something mechanical different than the standard medical hud which it does not look like it does.